### DISCUSSION TITLE
Broken links `https://xunit.github.io/xunit.analyzers/rules/<RULE>`

### DISCUSSION THREAD

**Discussion by norwd** [xunit/xunit#3043] [Feedback Request]
When building a unit test project with `xunit.analyzers`, the build output will include links in the form `https://xunit.github.io/xunit.analyzers/rules/<RULE>`, e.g. https://xunit.github.io/xunit.analyzers/rules/xUnit1026. However, this link results in a 404:

![image](https://github.com/user-attachments/assets/f6e84aed-96cb-40c0-a03b-fe01ac530052)

A quick look around shows that this should redirect to https://xunit.net/xunit.analyzers/rules/xUnit1026.

I think a good first step would be to update the console output to use the correct URL, but an even better solution would be to setup GitHub pages on the https://github.com/xunit/xunit.analyzers repo so that all the existing links no longer return 404. The pages site could even have a redirect in place so that https://xunit.github.io/xunit.analyzers/rules/xUnit1026 automatically redirects to https://xunit.net/xunit.analyzers/rules/xUnit1026.

**Comment by bradwilson** [comment]
These links are embedded in the analyzers. The correct links were updated in June of 2020 and have been available since `xunit.analyzers` version `1.0.0` (released in August 2022).

Please upgrade your version of `xunit.analyzers`.
